### PR TITLE
adding -y

### PR DESCRIPTION
**Pull request by chrislovecnm**
/assign @andrewsykim @geojaz

### CODE REVIEW

**Comment by andrewsykim** [comment]
Should we add an equivalent for update cluster?

**Comment by justinsb** [comment]
/lgtm

The conflict means this LGTM won't do any good though :-(  But feel free to rebase and self-lgtm

And I certainly agree with adding it to other places as well @andrewsykim

**Comment by chrislovecnm** [comment]
@andrewsykim PTAL

File: cmd/kops/update_federation.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ var (
 
 	update_federation_example = templates.Examples(i18n.T(`
 	# After cluster has been edited or upgraded, configure it with:
-	kops update federation k8s-cluster.example.com --yes --state=s3://kops-state-1234 --yes
```
Comment (andrewsykim) [inline cmd/kops/update_federation.go]: Did you mean to keep at least one `--yes`?

File: docs/cli/kops_update_federation.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ kops update federation
 
 ```
   # After cluster has been edited or upgraded, configure it with:
-  kops update federation k8s-cluster.example.com --yes --state=s3://kops-state-1234 --yes
```
Comment (andrewsykim) [inline docs/cli/kops_update_federation.md]: same here, did you mean to leave 1 `--yes`?

File: cmd/kops/update_federation.go (hunk 1 above)
Comment (chrislovecnm) [reply cmd/kops/update_federation.go]: There is no --yes on the options object unless I missed it

**Pull request merged**